src/watchman/models.clj

```diff
@@ -206,12 +218,36 @@
     (k/set-fields fields)
     (k/where {:id id})))
 
+(defn snooze-msg [snooze-until]
+  "Given a Joda datetime, returns a formatted message for the UI indicating until when
+  this role is snoozed"
```

Comment:
This isn't line-wrapped at 110 chars. End sentences with periods. See the rest of the file for the indentation used on multi-line comments.